.github/CODEOWNERS

```
@@ -6,24 +6,30 @@
 ###########
 
 # Catch all for non-code project files and unowned files | folders
-*      @ahsonkhan @rickwinter @joshfree
-/sdk/  @ahsonkhan @rickwinter @joshfree
+*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989
+/sdk/                    @rickwinter @ahsonkhan @antkmsft @vhvb1989
 
 # Samples
-/samples/      @ahsonkhan @rickwinter
-/sdk/samples/  @ahsonkhan @rickwinter
+/samples/                @rickwinter @ahsonkhan @antkmsft @vhvb1989
 
-# Azure::Core
-/sdk/core/     @ahsonkhan @antkmsft @rickwinter @vhvb1989
+# PRLabel: %Azure.Core
+/sdk/core/               @rickwinter @ahsonkhan @antkmsft @vhvb1989
 
-# Azure::Identity
-/sdk/identity/     @antkmsft
+# PRLabel: %Azure.Identity
+/sdk/identity/           @antkmsft @schaabs @ahsonkhan @rickwinter @vhvb1989
 
-# Service teams
-/sdk/storage/  @vinjiang @katmsft @JinmingHu-MSFT @antkmsft @rickwinter @vhvb1989
+# PRLabel: %KeyVault
+/sdk/keyvault/           @vhvb1989 @ahsonkhan @antkmsft @rickwinter
+
+# PRLabel: %Storage
+/sdk/storage/            @vinjiang @katmsft @JinmingHu-MSFT @ahsonkhan @antkmsft @rickwinter @vhvb1989
```

Please remove me from the **default** reviewer for this. It floods my notifications and I prefer getting tagged explicitly for something that I can help review or poll the PR list myself, as needed.

That's why I changed it in https://github.com/Azure/azure-sdk-for-cpp/pull/728

```suggestion
/sdk/storage/            @vinjiang @katmsft @JinmingHu-MSFT @antkmsft @rickwinter @vhvb1989
```